Source/DafnyCore/Dafny.atg

```diff
@@ -3650,7 +3650,7 @@ AssertStmt<out Statement/*!*/ s, bool inExprContext>
   .)
   "assert"                                     (. x = t; startToken = t; .)
   { Attribute<ref attrs> }
-  ( [ IF(IsLabel(!inExprContext))
+  ( [ IF(IsLabel(true))
```

Comment:
The `inExprContext` parameter of production `AssertStmt` is no longer needed and can be removed.